dash_core_components/package.json

```diff
@@ -33,6 +34,7 @@
     "babel-preset-env": "^1.7.0",
     "babel-preset-react": "^6.24.1",
     "builder": "3.2.2",
+    "color": "^3.1.0",
```

Comment:
New dependency used for darkening hex colors, used in the `CubeSpinner` component. It allows users to provide a color, and the component will figure out the appropriate darker colors that are used in the cube.